content/en/real_user_monitoring/browser/_index.md

```diff
@@ -407,7 +424,8 @@ Add the generated code snippet to the head tag of every HTML page you want to mo
       trackResources: true,
       trackLongTasks: true,
       trackInteractions: true,
-    })
+    });
+   DD_RUM.startSessionReplayRecording();
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**❗ warning:** ‏ ‏CDN async > before 4.30.0 > us5 is missing the startSessionReplayRecording